client/src/App.js

```diff
@@ -2,17 +2,33 @@ import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+// User Context Information
+import { UserContext } from "./context/UserContext";
+import useGetUserData from "./hook/useGetUserData";
```

Comment:
I ran into some issues running this because there was a folder called hooks, but this is looking for a folder called hook (without the S)

Reply:
sorry about that.  Windows does not see the difference between Hooks and hooks, so my fix to was to rename it "hook" in the meaentime.  I got a bit distracted between the pushes.  Sorry about the error. It should be fixed now.